src/site/content/en/accessible/content-reordering/index.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: Content re-ordering
+authors:
+  - rachelandrew
+date: 2020-05-28
+description: |
+  When creating a layout using CSS, you need to take care that you don't create a disconnected experience for users who are navigation with the keyboard.
+tags:
+  - blog
+  - accessibility
+---
+
+The order of content in your document is important for the accessibility of your site.
+A screen reader will read out content based on the document order,
+using the HTML elements that you have selected to give meaning to that content.
+A person navigating the site using a keyboard, rather than a touchscreen or mouse, will tab around the document.
+This means that they will jump from active element to active element,
+tabbing between links and form fields, once again in the order they exist in the document.
+
+Starting with a well-structured document and using all of the right HTML elements
+is therefore a key part of creating an accessible site.
+However, it is possible to undo some of that good work when you start using CSS. Let's take a look at why.
+
+## Source vs. visual order
+
+Website navigation is often marked up as a list of links.
+You can then use Flexbox to turn these into a horizontal bar.
+In the Glitch example below, I have created this commonly used pattern.
+Click into the example, and tab between the links.
+The focus will move in a logical direction from left to right,
+the order that we read in English.
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/flex-nav-source-ordered?path=index.html&previewSize=100"
+    title="flex-nav-source-ordered on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+If you have created this sort of pattern and then were asked to move "Contact Us",
+which is second in the source, to the end. You could use the `order` property which works in Flexbox.
+Try tabbing through the items in the example below, which has used the `order` property to rearrange the items.
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/flex-nav-ordered?path=style.css&previewSize=100"
+    title="flex-nav-ordered on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+The focus jumps to the final item and then back again.
+As far as the tab order is concerned that item is the second item.
+Visually however, it's the last item.
+
+The above example highlights the problem that we face if we rearrange and reorder content using CSS.
+If you were dealing with this issue then the right thing to do would be to change the order in the source,
+rather than using CSS.
+
+## Which CSS properties can cause reordering?
+
+Any layout method that allows you to move elements around can cause this problem.
+The following list being key places to consider if you are causing the problem.
```

Comment:
I found the wording on these two sentences a little hard to grok. I read it a couple times to understand it. Might be worth rephrasing somehow?

Reply:
```suggestion
Any layout method that allows you to move elements around can cause this problem.
The following CSS properties commonly cause content reordering problems:
```